sonar-project.properties

```diff
@@ -11,4 +11,4 @@ sonar.go.coverage.reportPaths = coverage.out
 sonar.go.tests.reportPaths = sonar-report.json
 sonar.tests = .
 sonar.test.inclusions = **/*_test.go
-sonar.coverage.exclusions = test/**/*,_examples/**/*
+sonar.coverage.exclusions = test/**/*,_examples/**/*,src/digitalidentity/**
```

Comment:
I am not sure this is the right approach. How did the other SDKs solve it?

Reply:
We should only exclude digitalidentity only from code duplication checks (so use sonar.cpd.exclusions)